src/commands/scaling/addScaleRule/addScaleRule.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizard, IActionContext, IWizardOptions } from "@microsoft/vscode-azext-utils";
+import { ext } from "../../../extensionVariables";
+import { ScaleRuleGroupTreeItem } from "../../../tree/ScaleRuleGroupTreeItem";
+import { localize } from "../../../utils/localize";
+import { AddNewScaleRule } from "./AddNewScaleRule";
+import { GetScaleRuleNameStep } from "./GetScaleRuleNameStep";
+import { GetScaleRuleTypeStep } from "./GetScaleRuleTypeStep";
+import { IAddScaleRuleWizardContext } from "./IAddScaleRuleWizardContext";
+
+export async function addScaleRule(context: IActionContext, node?: ScaleRuleGroupTreeItem): Promise<void> {
+    if (!node) {
+        node = await ext.tree.showTreeItemPicker<ScaleRuleGroupTreeItem>(new RegExp(ScaleRuleGroupTreeItem.contextValue), context);
+    }
+
+    const title: string = localize('addScaleRuleTitle', 'Create Scale Rule');
```

Comment:
Kind of weird that it's called "Add Scale Rule" everywhere, but the title is "Create Scale Rule".

Reply:
Yeah, I actually consciously made that choice just because it felt like a more accurate description of those specific steps, but I totally see why the lack of consistency would feel weird so I'll change it